bindings/experimental/distrdf/test/test_buildranges.py

```diff
@@ -208,42 +211,17 @@ def test_clustered_ranges_with_many_clusters_four_partitions(self):
         headnode.npartitions = 4
 
         crs = headnode.build_ranges()
-        ranges = rangesToTuples(crs)
+        ranges = treeranges_to_tuples(crs)
 
         ranges_reqd = [
-            (0, 250),
-            (250, 500),
-            (500, 750),
-            (750, 1000)
+            (0, 250, ["backend/1000clusters.root"]),
+            (250, 500, ["backend/1000clusters.root"]),
+            (500, 750, ["backend/1000clusters.root"]),
+            (750, 1000, ["backend/1000clusters.root"])
         ]
 
         self.assertListEqual(ranges, ranges_reqd)
 
-    def test_clustered_ranges_with_many_clusters_many_partitions(self):
-        """
-        Check that _get_clustered_ranges creates clustered ranges as equal as
-        possible for the maximum number of possible partitions (number of
-        clusters)
-
-        """
-
-        treename = "myTree"
-        filelist = ["backend/1000clusters.root"]
-        headnode = create_dummy_headnode(treename, filelist)
-        headnode.npartitions = 1000
-
-        crs = headnode.build_ranges()
-        ranges = rangesToTuples(crs)
-
-        start = 0
-        end = 1000
-        step = 1
-
-        ranges_reqd = [(a, b) for a, b in zip(range(start, end, step),
-                                              range(step, end + 1, step))]
-
-        self.assertListEqual(ranges, ranges_reqd)
-
```

Comment:
why is this test removed by the first commit in the PR?

Reply:
It's exactly the same as the following `test_buildranges_with_clustered_ranges` . Maybe at some point in the past they were testing different things, but I doubt it since `build_ranges` calls anyway `get_clustered_ranges`. Dataset, number of partitions etc. are all the same in the two tests so I removed one of them

Reply:
ok but then this removal doesn't have anything to do with the commit or the PR :smile:

Reply:
Ok I will revert this change